pkgs/development/ruby-modules/rbenv/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "rbenv";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    owner = "rbenv";
+    repo = "rbenv";
+    rev = "v${version}";
+    sha256 = "12i050vs35iiblxga43zrj7xwbaisv3mq55y9ikagkr8pj1vmq53";
+  };
+
+  postPatch = ''
+     patchShebangs src/configure
+     cd src
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv libexec completions $out
```

Comment:
I don't think the completions are copied to the right directory.
if you add `installShellFiles` to `nativeBuildInputs` than you can do:

```
installShellCompletion completions/rbenv.{bash,zsh}
```

and it does the right thing (tm)